.github/workflows/ci.yml

```diff
@@ -115,10 +115,13 @@ jobs:
           poetry install -E webgl_renderer
 
       - name: Run tests
-        run: poetry run pytest
+        run: poetry run coverage run -m pytest
 
       - name: Run module doctests
-        run: poetry run pytest --doctest-modules manim
+        run: poetry run coverage run --append -m pytest --doctest-modules manim
```

Comment:
I would like to use pytest-cov https://pypi.org/project/pytest-cov/ plugin here rather than running like this, and add to pytest config so that it passes `--cov` when pytest is called by default.

Reply:
Sounds good. I'm currently in the middle of setting this up and figuring out the right configs